app/routes/docs.circulars.archive.mdx

```diff
@@ -17,7 +19,32 @@ Upon successful submission and distribution, a GCN Circular is assigned a number
 
 You can search the Circulars Archive by keywords or phrases in the subject, body, and submitter fields. By default, results will be returned in reverse chronological order, but you can also choose to sort the results by relevance. You can navigate through the search results using pagination controls and specify the number of results to display per page. By default, 100 results are displayed per page.
 
-You can also filter search results by submission date. Select the dropdown button labeled "Filter by date". From there, you can either select from a number of predefined time ranges (e.g., "Last Hour", "Last Day", etc.) or specify a custom date range, which can either be set as a start date, an end date, or both. All search filters can be cleared by clicking the 'x' button in the search bar.
+You can filter search results by submission date. Select the dropdown button labeled "Filter by date". From there, you can either select from a number of predefined time ranges (e.g., "Last Hour", "Last Day", etc.) or specify a custom date range, which can either be set as a start date, an end date, or both. All search filters can be cleared by clicking the 'x' button in the search bar.
+
+You can enter a specific Circular ID (e.g. 12345) and it will list that Circular. Note that GRB names without a letter can be interpreted as Circular IDs.
```

Comment:
This doesn't depend on synonyms, does it? Would you please pull this out as a separate PR?